CMake/resolve_dependency_modules/README.md

```diff
@@ -32,11 +32,11 @@ by Velox. See details on bundling below.
 | re2               | 2021-04-01      | Yes      |
 | fmt               | 10.1.1          | Yes      |
 | simdjson          | 3.9.3           | Yes      |
-| folly             | v2024.07.01.00  | Yes      |
-| fizz              | v2024.07.01.00  | No       |
-| wangle            | v2024.07.01.00  | No       |
-| mvfst             | v2024.07.01.00  | No       |
-| fbthrift          | v2024.07.01.00  | No       |
+| folly             | v2024.07.15.00  | Yes      |
+| fizz              | v2024.07.15.00  | No       |
+| wangle            | v2024.07.15.00  | No       |
+| mvfst             | v2024.07.15.00  | No       |
+| fbthrift          | v2024.07.15.00  | No       |
```

Comment:
@kgpai I didn't see where the versions are defined for `fizz`, `wangle`, `mvfst`, or `fbthrift`. Are those a part of Folly? It seems like they are tied to the same version.

Reply:
They all share the same version : FB_OS_VERSION .